Sources/Gravatar/Identifiers/ProfileIdentifier.swift

```diff
@@ -37,4 +37,14 @@ extension ProfileIdentifier: Identifiable {
             hashID.id
         }
     }
+
+    // Returns the `AvatarIdentifier` popuplated from `self`.
+    public var avatarIdentifier: AvatarIdentifier {
+        switch self {
+        case .email(let email):
+            AvatarIdentifier.email(email.string)
+        case .hashID(let hashID):
+            AvatarIdentifier.hashID(hashID.string)
+        }
```

Comment:
```suggestion
        AvatarIdentifier.hashID(id)
```

I believe this could be simplified like this 🤔 
The switch is already performed by the `id` var

Reply:
It would be shorter for sure but I thought it would be good to keep the original character of the id, like if there's an email involved we still keep it in the `.email` enum. 🤔

Reply:
Oh I see. As we shouldn't need the literal email it shouldn't really matter, right? 
But not big feelings about this 👍 
Feel free to keep it as it is

Reply:
Nope we don't need the email directly and things should work with just hash. I just had a hunch like, as a 3rd party dev if i created the profile identifier with `.email(...)` then I would expect that this conversion respects my choice.